app/listeners/action_cable_listener.rb

```diff
@@ -156,6 +156,14 @@ def conversation_mentioned(event)
     broadcast(account, [user.pubsub_token], CONVERSATION_MENTIONED, conversation.push_event_data)
   end
 
+  def custom_filter_updated(event)
+    custom_filter = event.data[:custom_filter]
+    account = custom_filter.account
+    tokens = user_tokens(account, account.agents)
+
+    broadcast(account, tokens, CUSTOM_FILTER_UPDATED, custom_filter.push_event_data)
+  end
+
```

Comment:
specs for the `custom_filter_update` event in action cable listener
Also, since the filters are private you only need to broadcast the event to the filter owner right?

Reply:
> Also, since the filters are private you only need to broadcast the event to the filter owner right?

Yes

Reply:
> Also, since the filters are private you only need to broadcast the event to the filter owner right?

@sojan-official Sending user access token, wouldn't help here?